rs/http_endpoints/public/src/metrics.rs

```diff
@@ -90,6 +91,10 @@ impl HttpHandlerMetrics {
                 // 1ms, 2ms, 5ms, 10ms, 20ms, ..., 10s, 20s, 50s
                 &REQUESTS_LABEL_NAMES,
             ),
+            in_flight_requests: metrics_registry.int_gauge(
```

Comment:
Using a gauge for something that changes often can be very confusing when looking at the dashboard because the gauge is scraped rarely so looking at the dashboard can be very misleading